docs/en/setup/backend/configuration-vocabulary.md

```diff
@@ -1,297 +1,296 @@
+| receiver-otel | default | Read [receiver doc](backend-receivers.md) for more details. | - | - |
+| - | - | enabledHandlers| Enabled handlers for otel. | SW_OTEL_RECEIVER_ENABLED_HANDLERS | - |
+| - | - | enabledOcRules| Enabled metric rules for OC handler. | SW_OTEL_RECEIVER_ENABLED_OC_RULES | - |
+| receiver_zipkin |default| Read [receiver doc](backend-receivers.md). | - | - |
+| - | - | restHost| Binding IP of RESTful services. |SW_RECEIVER_ZIPKIN_HOST|0.0.0.0|
+| - | - | restPort | Binding port of RESTful services. | SW_RECEIVER_ZIPKIN_PORT|9411|
+| - | - | restContextPath| Web context path of RESTful services. | SW_RECEIVER_ZIPKIN_CONTEXT_PATH|/|
+| receiver_jaeger | default| Read [receiver doc](backend-receivers.md). | - | - |
+| - | - | gRPCHost|Binding IP of gRPC services. Services include gRPC data report and internal communication among OAP nodes. | SW_RECEIVER_JAEGER_HOST | - |
+| - | - | gRPCPort| Binding port of gRPC services. | SW_RECEIVER_JAEGER_PORT | - |
+| - | - | gRPCThreadPoolSize| Pool size of gRPC server. | - | CPU core * 4|
+| - | - | gRPCThreadPoolQueueSize| Queue size of gRPC server. | - | 10000|
 | - | - | maxConcurrentCallsPerConnection | The maximum number of concurrent calls permitted for each incoming connection. Defaults to no limit. | - | - |
-| - | - | maxMessageSize | Sets the maximum message size allowed to be received on the server. Empty means 4 MiB | - | 4M(based on Netty) |
-| prometheus-fetcher | default | Read [fetcher doc](backend-fetcher.md) for more details | - | - |
-| - | - | enabledRules | Enable rules. | SW_PROMETHEUS_FETCHER_ENABLED_RULES | self |
+| - | - | maxMessageSize | Sets the maximum message size allowed to be received on the server. Empty means 4 MiB. | - | 4M(based on Netty) |
+| prometheus-fetcher | default | Read [fetcher doc](backend-fetcher.md) for more details. | - | - |
+| - | - | enabledRules | Enabled rules. | SW_PROMETHEUS_FETCHER_ENABLED_RULES | self |
 | - | - | maxConvertWorker | The maximize meter convert worker. | SW_PROMETHEUS_FETCHER_NUM_CONVERT_WORKER | -1(by default, half the number of CPU core(s)) |   
-| kafka-fetcher | default | Read [fetcher doc](backend-fetcher.md) for more details | - | - |
+| kafka-fetcher | default | Read [fetcher doc](backend-fetcher.md) for more details. | - | - |
 | - | - | bootstrapServers | A list of host/port pairs to use for establishing the initial connection to the Kafka cluster. | SW_KAFKA_FETCHER_SERVERS | localhost:9092 |
-| - | - | namespace | namespace aims to isolate multi OAP cluster when using the same Kafka cluster.if you set a namespace for Kafka fetcher, OAP will add a prefix to topic name. you should also set namespace in `agent.config`, the property named| SW_NAMESPACE | - |
```

Comment:
Incomplete sentence. Is something missing here?

Reply:
the property is named `plugin.kafka.namespace`